src/common/ant.ts

```diff
@@ -227,14 +227,14 @@ export class AoANTReadable implements AoANTRead {
 }
 
 export class AoANTWriteable extends AoANTReadable implements AoANTWrite {
-  private signer: ContractSigner;
+  private signer: AoSigningFuction;
```

Comment:
might be good to rename "signer" to "aoSigner" through the SDK to make it self-documenting

Reply:
i prefer `signer` to keep it abstracted from it's type, but a nit